.gitignore

```diff
@@ -9,14 +9,6 @@ _site
 # Temporary files
 tmp
 
-# Raster graphics
-*.jpg
-*.jpeg
```

Comment:
@thomasjbradley I just want to confirm if allowing image uploads is okay for this repository, or if you have a different protocol for that.